WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -21,17 +53,107 @@ extension UIImage {
         return Gridicon.iconOfType(.chevronRight).imageWithTintColor(tintColor)!
     }
 
+    /// Chevron pointing down
```

Comment:
To keep the comments consistent, would you Title Case all the icon and image names? This one would be `Chevron Pointing Down`